Library/Homebrew/os/linux/global.rb

```diff
@@ -1,5 +1,8 @@
 # frozen_string_literal: true
 
+HOMEBREW_READELF_RB = ENV["HOMEBREW_READELF_RB"].present?.freeze
```

Comment:
Let's add a short comment above this line describing what these two feature flags do.

Reply:
i wonder if a single flag would suffice. The individual flags are kept for future purpose, in case to enable patchelf  or readelf only changes.

And I'll comment as requested

Reply:
Yeah, I think a single `HOMEBREW_PATCHELF_RB` feature flag would be sufficient.

Reply:
Agreed, and done.